src/anemoi/graphs/create.py

```diff
@@ -93,6 +98,9 @@ def _path_readable(self) -> bool:
         """Check if the output path is readable."""
         import torch
 
+        if self.path is None:
```

Comment:
Should this be part of _path_readable()?

Reply:
Where else would you put it?

Reply:
If there is no path, you don't need to check whether the file is readable.